module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -171,7 +171,7 @@ protected function pickValue($rawList, $mode, $msg, $transPrefix = false)
         } elseif (count($list) == 0) {
             // Empty list?  Return a blank string:
             return '';
-        } elseif ($mode == 'all') {
+        } elseif ($mode == 'all' || $mode == 'group') {
```

Comment:
Why is this change necessary? I'm just trying to figure out how it relates to everything else. According to the function comment, 'group' is not a $mode value that this method is expecting... but maybe the comment is wrong. :-)

Reply:
Ah yes, I meant to call this out and then forgot to. I was going to ask you about this. I felt like this might be fixing an unrelated bug but I wasn't sure. I might have misunderstood what I was seeing. Without this change I was getting an "Undefined" message (or something to that effect) on some holdings with multiple call numbers. This is probably unrelated to what I'm doing with the rest of the PR but I left it so we could have this conversation. From my point of view, it's fine to remove this but if you want me to leave it, that's fine to.

Reply:
Thanks, @bbusenius. If you can remember what setup caused the error, that would be helpful, but if not, I'll do some experimentation and see if I can get to the bottom of whatever is going on. If this truly is a bug (which seems entirely possible), maybe I can open a separate PR to deal with that issue, add tests to reproduce it, etc.

Reply:
No problem. When I remove `|| $mode == 'group'` and configure it like this:
```
multiple_call_nos = group
multiple_locations = group
```
I get something that looks like this:

![image](https://user-images.githubusercontent.com/7826429/120714197-84f24b80-c488-11eb-91d7-de67db605554.png)

When I add the `||` clause above, I get the following result which looked more correct to me:

![image](https://user-images.githubusercontent.com/7826429/120714601-05b14780-c489-11eb-9aba-c7ffd1d741ca.png)

This particular record is pretty crazy but I think this is right. There were lots of others when I did this though. Anyhow, we don't use this configuration and I don't know how common this would be in practice.

Reply:
Thanks, @bbusenius, that's very helpful! I'm in a conference all day today, but I'll take a closer look early next week.

Reply:
@bbusenius, I took a closer look at this, and I think it's an unnecessary change: the [documentation in the .ini file](https://github.com/vufind-org/vufind/blob/dev/config/vufind/config.ini#L1726-L1727) indicates that the "group" setting is only valid for the multiple_locations setting. It is not supported for multiple_call_nos. Perhaps we should actually change this so that an exception is thrown if you provide an unsupported option for this setting. What do you think?

Reply:
Oh right. Gosh, now that you mention it, I remember reading that at some point. I think I might have forgotten after I let this sit for awhile. I reverted that part of the code.

I'm agnostic about the exception. I'm not sure if that would be helpful or not. I'm happy to do it if you think it would be best though.

Reply:
The exception might have saved you the trouble of digging into this in the first place, but other than that, I think the benefit is pretty limited. I'm happy to just leave it alone for now. :-)